lib/ansible/utils/hashing.py

```diff
@@ -93,3 +93,6 @@ def md5(filename):
         raise ValueError('MD5 not available.  Possibly running in FIPS mode')
     return secure_hash(filename, _md5)
 
+from binascii import crc32 as crc32
```

Comment:
Only need to import binascii module here and it should be near the top of the file.